src/web/handlers/FlingGestureHandler.ts

```diff
@@ -51,21 +48,31 @@ export default class FlingGestureHandler extends GestureHandler {
   }
 
   private tryEndFling(): boolean {
+    const velocityVector = new Vector().fromVelocity(
+      this.tracker,
+      this.keyPointer
+    );
+
+    const getAlignment = (direction: Directions) => {
+      const directionVector = DirectionTypeVectorMappings.get(direction);
+      return (
+        directionVector &&
+        direction & this.direction &&
+        velocityVector.isSimilar(directionVector, this.minDirectionalAlignment)
+      );
```

Comment:
```suggestion
      const directionVector = DirectionTypeVectorMappings.get(direction)!;
      return (
        direction & this.direction &&
        velocityVector.isSimilar(directionVector, this.minDirectionalAlignment)
      );
```

Similar to the situation in `Vector.ts`, as far as I can see `directionVector` will never be undefined, so we can use non-null assertion (correct me if I'm wrong).